blocks/form/form.js

```diff
@@ -108,4 +108,9 @@ export default async function decorate(block) {
       }
     }
   });
+  // Logic for containerizing all radio buttons DIV in one container for Header form
```

Comment:
fieldsets should be able to cover this need (and the "heading-wrapper" query looks like it's failing)